.travis.yml

```diff
@@ -36,4 +37,6 @@ before_script:
   - npm run link-daemon ./daemon-binary
 
 script:
+  - npx ganache-cli --account="0x1f0d511e990ddbfec302e266d62542384f755f6cc6b3161b2e49a2a4e6c4be3d,100000000000000000000" > /dev/null &
+  - sleep 5
```

Comment:
Can we do something better than sleep here? We're at the mercy of more or less uncontrolled machines for CI, so we don't have a guaranteed bound on the time to start.

A script wrapping `expect` is a heavy, but potentially feasible option.

Reply:
Agreed that sleeping is not ideal. I'm not familiar with `expect`, just looked it up. Can I basically just do this in a script?
```
npm run deploy-ganache &
expect eof
```

Reply:
I have only basic familiarity too FWIW. I'll get back to you on that. It should be the last resort option anyhow.